tests/test_macosx_libfile.py

```diff
@@ -20,17 +20,23 @@ def test_read_from_dynlib():
         ("test_lib_10_6_386.dylib", "10.6"),
         ("test_lib_10_10_386.dylib", "10.10"),
         ("test_lib_10_14_386.dylib", "10.14"),
-        ("test_lib_multiple_fat.dylib", "10.14")
+        ("test_lib_multiple_fat.dylib", "10.14"),
```

Comment:
IMO, I think it would be clearer to add the patch version in to this list, since that's what the extract_macosx_min_system_version function is returning

Reply:
i.e.

```
        ("test_lib_multiple_fat.dylib", "10.14.0"),
```
and the same for the others

Reply:
ok